company-nxml.el

```diff
@@ -129,13 +219,29 @@
                 (company-nxml-attribute 'prefix)
                 (company-nxml-attribute-value 'prefix)))
     (candidates (cond
+                 ((and (eq company-nxml-last-command 'company-nxml-tag)
+                       (company-grab company-nxml-in-starttag-name-regexp 1 1))
+                  (setq company-prefix "") ; don't zap the just inserted tag name!
```

Comment:
This looks odd. Backends aren't supposed to modify `company-prefix`.

And do you absolutely require `company-nxml-last-command` variable? Is it impossible to distinguish via regexps, like the current code does, or by looking around?

We don't guarantee that the `prefix` command is always followed by the corresponding `candidates` command. Imagine e.g. `company-sort-by-occurrence` in combination with `company-search-toggle-filtering`.